js/src/components/PrimaryTaxonomyFilter.js

```diff
@@ -0,0 +1,81 @@
+/* global window */
+/* External dependencies */
+import React from "react";
+import PropTypes from "prop-types";
+import { Fragment } from "@wordpress/element";
+import get from "lodash/get";
+import values from "lodash/values";
+
+/* Internal dependencies */
+import PrimaryTaxonomyPicker from "./PrimaryTaxonomyPicker";
+
+let taxonomyData = null,
+	taxonomiesWithPrimaryTermSupport = null;
+
+class PrimaryTaxonomyFilter extends React.Component {
+	constructor( props ) {
+		super( props );
```

Comment:
minor: technically, passing props in this case is redundant, as in teh consstructor there's no use of props or this.props